gtfs/spec/en/reference.md

```diff
@@ -436,6 +436,7 @@ To process the cost of a leg:
 |  `end_timeframe_group_id` | Foreign ID referencing `timeframes.timeframe_group_id` | Optional |  Defines the timeframe for the arrival of the fare leg.<br><br>For a fare leg rule that specifies a `end_timeframe_group_id`, that rule will match a particular leg if there exists at least a row in `timeframes.txt` where all of the following conditions are true:<br>- The row’s `timeframe_group_id` is equal to the `end_timeframe_group_id` value.<br>- The set of service days identified by the row’s `service_id` contains the service day of the leg’s departure trip.<br>- The arrival time of the leg, relative to the service day per [GTFS Time field type](#field-types) conventions, is greater than or equal to the rows `start_time` value and less than the `end_time` value.<br><br>If there are no matching `fare_leg_rules.end_timeframe_group_id` values to the `timeframe_group_id` being filtered, empty `fare_leg_rules.end_timeframe_group_id` will be matched by default. <br><br>An empty `fare_leg_rules.end_timeframe_group_id` indicates that the end time of the leg does not affect the fare. |
 | `timeframe_type` | Enum | **Conditionally Required** | Defines the type of fare validation to be associated with `timeframes.start_time` and `timeframes.end_time`.<br><br>Valid options are:<br>`0` - In vehicle.<br>`1` - At fare gate or on platform.<br><br>Conditionally Required:<br>- **Required**  if either `start_timeframe_group_id` or `end_timeframe_group_id` is defined.<br>- **Forbidden** otherwise.|
 | `fare_product_id` | Foreign ID referencing `fare_products.fare_product_id` | **Required** | The fare product required to travel the leg. |
+| `override` | Non-negative integer | Optional | When multiple entries in `fare_leg_rules.txt` match, the one with the highest `override` value will be selected.<br><br>An empty value for `override` is treated as zero.|
```

Comment:
I think this should be "the **ones** with the highest" - there's no guarantee that there will be only a single highest value. It might need extra words to make that clear.

Reply:
I took a stab at improving the description of override in [this commit](https://github.com/google/transit/pull/357/commits/d4afc6ede5eae96567b373a9400d6f0c2e907442). Let me know if this is easier to interpret.